tests/web3js/eth_streaming_test.js

```diff
@@ -20,7 +20,7 @@ it('streaming of logs using filters', async() => {
         { A: repeatA, B: 400 },
     ]
 
-    let ws = new Web3("ws://localhost:8545")
+    let ws = new Web3("ws://127.0.0.1:8545")
```

Comment:
This change is needed because the GH test runner probably uses IPv4, instead of IPv6.